moabb/paradigms/motor_imagery.py

```diff
@@ -69,21 +69,22 @@ def process_raw(self, raw, dataset):
 
         # get event id
         if self.events is None:
-            event_id = dataset.event_id
+            event_id = dataset.selected_events
         else:
             event_id = {ev: dataset.event_id[ev] for ev in self.events}
 
         # pick events, based on event_id
         try:
             events = mne.pick_events(events, include=list(event_id.values()))
-        except RuntimeError:
+        except RuntimeError as r:
             # skip raw if no event found
             return
 
         # get interval
         if self.interval is None:
             tmin, tmax = dataset.interval
         else:
+            # TODO: make this work with varying offsets since not all data starts at 0
```

Comment:
lets use `epochs.crop()` after epoching

Reply:
what if someone wants to do a baseline though? Wouldn't it be better to define the window while making the Epochs object, instead of making it with the full task interval and then cropping it after?